src/binary/ibc-setup/commands/init.ts

```diff
@@ -12,7 +14,41 @@ export type Options = GlobalOptions & {
   readonly dest: string;
 };
 
+type Registry = {
+  version: string;
+  chains: Record<
+    string,
+    {
```

Comment:
Can you declare a separate (non-inline) type for this? eg. `chains: Record<string, ChainInfo>` and then that last part as ChainInfo (I assume we will just want that in many places)

Reply:
51b003c